appveyor.yml

```diff
@@ -1,73 +0,0 @@
-version: 1.0.{build}
```

Comment:
I think we should rename appveyor-gn.yml to appveyor.yml.  We may want to do that after this PR is merged though.

Reply:
> We may want to do that after this PR is merged though.

That was exactly the plan )